crates/serpen/tests/fixtures/ast_rewriting/alias_transformation_test/utils/helpers.py

```
@@ -0,0 +1,19 @@
+"""
+Helper utilities module.
+Used to test mixed from-import scenarios (some aliased, some not).
+"""
+
+
+def helper_func(input_str):
```

Ruff (0.11.9)

Missing return type annotation for public function `helper_func`

Add return type annotation: `str`

(ANN201)